### PR TITLE
feat: style import button

### PR DESCRIPTION
**Pull request by betodealmeida**
### SUMMARY

Use the antd `Upload` component when importing models. This allows us to style the button and have slightly better UX.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

Before:

Screenshot 2021-06-08 at 22-28-49 Superset

After:

https://user-images.githubusercontent.com/1534870/121298994-9e8cfc00-c8a9-11eb-9abe-1caee071ba14.mov

@yousoph @Steejay what do you think of the new modal?

### TESTING INSTRUCTIONS

Imported database manually.

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by yousoph** [comment]
ooh loving how the file name looks after it's selected!

File: superset-frontend/src/components/ImportModal/index.tsx [hunk 1]
```diff
@@ -16,10 +16,13 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import React, { FunctionComponent, useEffect, useRef, useState } from 'react';
+import React, { FunctionComponent, useEffect, useState } from 'react';
+import { Upload } from 'antd';
```
Comment (eschutho) [inline superset-frontend/src/components/ImportModal/index.tsx:20]: A few of us were just talking about antd imports the other day. Do you mind importing and exporting this component in the /superset-frontend/src/common/components/index.tsx file and pulling it in from there? We're going to try to clean all these up at some point for consistency.

Comment (betodealmeida) [reply superset-frontend/src/components/ImportModal/index.tsx:20]: Yeah, I like having that abstraction layer for the day we move from `antd` to the new fancy component library. Will move.

**Review by eschutho — APPROVED** [review]
This new button looks great!

**Comment by junlincc** [comment]
thanks for the improvement! is there any validation for invalid upload? 
also the initial state with 3 buttons(1 standalone on the higher left, 2 at the bottom) in the modal looks a bit off balance visually, a centered drag and drop area will be ideal, eventually. @steejay 

Screen Shot 2021-06-09 at 1 20 34 PM

**Comment by betodealmeida** [comment]
> thanks for the improvement! is there any validation for invalid upload?

There is, the backend does a variety of validations, including:

1. If the object exists, user needs to confirm overwrite.
2. If the DB requires a password, user is prompted for it.
3. Is the file format correct (ZIP, JSON, YAML, depending on the object)?
4. For ZIP, are the contents valid?

**Pull request merged**